### PR TITLE
Rewrites/curly braces is result usage 001

### PR DESCRIPTION
**Pull request by Quix0r**
This PR contains the following changes and is the first of the (rebooted) series of changes that are coming up:

- curly braces added
- empty lines added for better readability (lesser compacted)
- debug messages added
- spaces around dots added (strings)
- some TODOs added for type-hint rewrite

### CODE REVIEW

File: src/Protocol/DFRN.php [hunk 1]
```diff
@@ -177,8 +178,9 @@ public static function feed($dfrn_id, $owner_nick, $last_update, $direction = 0,
 			$groups = Group::getIdsByContactId($contact['id']);
 
 			if (count($groups)) {
-				for ($x = 0; $x < count($groups); $x ++)
+				for ($x = 0; $x < count($groups); $x ++) {
 					$groups[$x] = '<' . intval($groups[$x]) . '>' ;
+				}
```
Comment (MrPetovan) [inline src/Protocol/DFRN.php]: Can you please add a new line after this closing brace like you did with the `if () { ... }` blocks?

File: mod/share.php [hunk 2]
```diff
@@ -23,28 +23,29 @@ function share_init(App $a) {
 	} else {
 		$o = share_header($r[0]['author-name'], $r[0]['author-link'], $r[0]['author-avatar'], $r[0]['guid'], $r[0]['created'], $r[0]['plink']);
 
-		if ($r[0]['title'])
+		if ($r[0]['title']) {
 			$o .= '[b]'.$r[0]['title'].'[/b]'."\n";
+		}
```
Comment (MrPetovan) [inline mod/share.php]: Can you please add a new line after this closing brace like you did with the `if () { ... }` blocks?

**Comment by tobiasd** [comment]
Is code beautification a fix or a new feature? IOW should this go into develop or 2018.05-rc?

**Comment by MrPetovan** [comment]
Code beautification, it correctly goes to develop.

**Comment by Quix0r** [comment]
Thank you.

**Pull request merged**